### ISSUE TITLE
[Bug]: npx sb init sets up broken configuration

### ISSUE THREAD

**Issue by Lootwig** [storybookjs/storybook#20188]
### Describe the bug

After following https://storybook.js.org/blog/storybook-for-svelte/, when running `npm run storybook`, the command fails with the following output:

```
> storybook-ts@0.0.1 storybook
> start-storybook -p 6006

info @storybook/svelte v6.5.14
info
ERR! Error [ERR_REQUIRE_ESM]: require() of ES Module D:\Projects\contribute\storybook-ts\sve
lte.config.js from D:\Projects\contribute\storybook-ts\.storybook\main.cjs not supported.   
ERR! Instead change the require of svelte.config.js in D:\Projects\contribute\storybook-ts\.
storybook\main.cjs to a dynamic import() which is available in all CommonJS modules.        
ERR!     at Object.<anonymous> (D:\Projects\contribute\storybook-ts\.storybook\main.cjs:16:1
9)
ERR!     at interopRequireDefault (D:\Projects\contribute\storybook-ts\node_modules\@storybo
ok\core-common\dist\cjs\utils\interpret-require.js:64:16)
ERR!     at serverRequire (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core-
common\dist\cjs\utils\interpret-require.js:101:10)
ERR!     at getPreviewBuilder (D:\Projects\contribute\storybook-ts\node_modules\@storybook\c
ore-server\dist\cjs\utils\get-preview-builder.js:25:55)
ERR!     at buildDevStandalone (D:\Projects\contribute\storybook-ts\node_modules\@storybook\
core-server\dist\cjs\build-dev.js:106:71)
ERR!     at async buildDev (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core
-server\dist\cjs\build-dev.js:174:5)
ERR!  Error [ERR_REQUIRE_ESM]: require() of ES Module D:\Projects\contribute\storybook-ts\sv
elte.config.js from D:\Projects\contribute\storybook-ts\.storybook\main.cjs not supported.  
ERR! Instead change the require of svelte.config.js in D:\Projects\contribute\storybook-ts\.
storybook\main.cjs to a dynamic import() which is available in all CommonJS modules.        
ERR!     at Object.<anonymous> (D:\Projects\contribute\storybook-ts\.storybook\main.cjs:16:1
9)
ERR!     at interopRequireDefault (D:\Projects\contribute\storybook-ts\node_modules\@storybo
ok\core-common\dist\cjs\utils\interpret-require.js:64:16)
ERR!     at serverRequire (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core-
common\dist\cjs\utils\interpret-require.js:101:10)
ERR!     at getPreviewBuilder (D:\Projects\contribute\storybook-ts\node_modules\@storybook\c
ore-server\dist\cjs\utils\get-preview-builder.js:25:55)
ERR!     at buildDevStandalone (D:\Projects\contribute\storybook-ts\node_modules\@storybook\
core-server\dist\cjs\build-dev.js:106:71)
ERR!     at async buildDev (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core
-server\dist\cjs\build-dev.js:174:5) {
ERR!   code: 'ERR_REQUIRE_ESM'
ERR! }

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

D:\Projects\contribute\storybook-ts\.storybook\main.cjs:16
    "preprocess": require("../svelte.config.js").preprocess
                  ^

Error [ERR_REQUIRE_ESM]: require() of ES Module D:\Projects\contribute\storybook-ts\svelte.c
onfig.js from D:\Projects\contribute\storybook-ts\.storybook\main.cjs not supported.        
Instead change the require of svelte.config.js in D:\Projects\contribute\storybook-ts\.story
book\main.cjs to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (D:\Projects\contribute\storybook-ts\.storybook\main.cjs:16:19)   
    at interopRequireDefault (D:\Projects\contribute\storybook-ts\node_modules\@storybook\co
re-common\dist\cjs\utils\interpret-require.js:64:16)
    at serverRequire (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core-commo
n\dist\cjs\utils\interpret-require.js:101:10)
    at getFrameworkPackage (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core
-common\dist\cjs\presets.js:326:50)
    at loadAllPresets (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core-comm
on\dist\cjs\presets.js:356:26)
    at buildDev (D:\Projects\contribute\storybook-ts\node_modules\@storybook\core-server\dis
t\cjs\build-dev.js:209:50) {
  code: 'ERR_REQUIRE_ESM'
}

```

### To Reproduce

```shell
create svelte@1.0.0-next.579 error
cd error
npm install
yes | npx sb init
npm run storybook
```

https://github.com/Lootwig/storybook-ts is a repo set up like this.

### System

```shell
Need to install the following packages:
  sb@7.0.0-beta.2
Ok to proceed? (y) y
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#depre
cated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated        
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resol
ve#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated

Environment Info:

  System:
    OS: Windows 10 10.0.19044
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
  Binaries:
    Node: 16.16.0 - C:\Program Files\nodejs\node.EXE
    npm: 8.18.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 108.0.5359.71
    Edge: Spartan (44.19041.1266.0), Chromium (108.0.1462.42)
  npmPackages:
    @storybook/addon-actions: ^6.5.14 => 6.5.14
    @storybook/addon-essentials: ^6.5.14 => 6.5.14
    @storybook/addon-interactions: ^6.5.14 => 6.5.14
    @storybook/addon-links: ^6.5.14 => 6.5.14
    @storybook/builder-vite: ^0.2.5 => 0.2.5
    @storybook/svelte: ^6.5.14 => 6.5.14
    @storybook/testing-library: ^0.0.13 => 0.0.13

```
```


### Additional context

_No response_

**Comment by shilman** [comment]
Can you try the 7.0 beta? we've made a lot of improvements to svelte support there

```
npm sb@next init
```

**Comment by Lootwig** [comment]
```
% npm sb@next init
Unknown command: "sb@next"

To see a list of supported npm commands, run:
  npm help
```

Also, [instructions for installing 7.0](https://storybook.js.org/docs/7.0/svelte/get-started/install) look exactly the same as they do for [6.5](https://storybook.js.org/docs/svelte/get-started/install)

**Comment by Bastian** [comment]
I ran into the same problem when running `npx sb init`.

I can confirm that `npx storybook@next init` solved the problem (`storybook@7.0.0-beta.15`).

**Comment by IanVS** [comment]
@Lootwig you need to use `npx`, not `npm`.  

Thanks for double-checking, @Bastian!
